superset/tasks/schedules.py

```diff
@@ -258,7 +258,7 @@ def _get_slice_data(schedule):
 
     opener = urllib.request.build_opener()
     opener.addheaders.append(("Cookie", f"session={cookies['session']}"))
-    response = opener.open(slice_url)
+    response = opener.open(slice_url, timeout=config["SUPERSET_WEBSERVER_TIMEOUT"])
```

Comment:
Does this solve a problem that you're facing? Based on the documentation, leaving timeout unset never triggers a timeout: https://docs.python.org/3/library/socket.html?highlight=socket#socket.getdefaulttimeout This leaves me to conclude that the timeout is thrown once the webserver's own timeout is reached, rendering this timeout setting obsolete. Is this not the case?